rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -235,14 +222,14 @@ std::string SqliteStorage::get_storage_identifier() const
 
 std::string SqliteStorage::get_relative_path() const
 {
-  return database_name_;
+  return relative_path_;
 }
 
 rosbag2_storage::BagMetadata SqliteStorage::get_metadata()
```

Comment:
didn't we say that `get_metadata()` should not be implemented in the individual storage plugins any longer?

Reply:
Yes, I opened Issue #197 which includes removing `get_metadata()` from the storage plugins. I wanted to keep it separate from this PR since that's an API-breaking change.

Reply:
If you decide to do that in a follow PR, then we can't delete the tests for it though.

Reply:
Ok, I'll block this PR on that Issue and open up a new PR.